frappe_whatsapp/frappe_whatsapp/doctype/whatsapp_notification/whatsapp_notification.py

```diff
@@ -33,7 +33,7 @@ def send_scheduled_message(self) -> dict:
             fieldname='language_code'
         )
         if language_code:
-            for contact in self._contact_list:
+            for contact in self.contact_list:
```

Comment:
contact_list has to be added as filed in doctype

Reply:
There is no such field named _contact_list